fbchat/client.py

```diff
@@ -1141,7 +1141,8 @@ def quickReply(self, quick_reply, thread_id=None, thread_type=None):
         :raises: FBchatException if request failed
         """
         quick_reply.is_response = True
-        return self.send(Message(text=quick_reply.title, quick_replies=[quick_reply]))
+        if quick_reply.type == QuickReplyType.TEXT:
+            return self.send(Message(text=quick_reply.title, quick_replies=[quick_reply]))
```

Comment:
Why do we only allow textual quick replies? Can't we use the other ones as well?

Reply:
In any case, this behaviour should be documented instead of being an `if` statement

Reply:
For location quick reply I need a `sendLocation` method from #333, so I'll wait for merging these two PRs, and for email and phone quick replies I'll make methods to get them and commit it to this PR.

Reply:
Okay, that makes sense. We should merge #333 first then, and update the implementation before we merge this